mne/io/ctf/ctf.py

```diff
@@ -217,11 +217,11 @@ def _get_sample_info(fname, res4, system_clock):
     else:
         n_trial = n_samp // res4['nsamp']
         n_omit = n_samp_tot - n_samp
-        n_samp = n_trial * res4['nsamp']
```

Comment:
I suspect that breaking this relationship could eventually break some code downstream. Can you try getting rid of `n_trial` and `n_samp_tot` from the output dict, and see if the downstream code still works or can be refactored to work? That would show places where breaking this relationship could be problematic.